pkg/controller/plan/kubevirt.go

```diff
@@ -784,8 +793,14 @@ func (r *KubeVirt) EnsureGuestConversionPod(vm *plan.VMStatus, vmCr *VirtualMach
 		err = liberr.Wrap(err)
 		return
 	}
+
+	for _, pod := range list.Items {
+		if !strings.Contains(pod.Name, "pvcinit") {
+			return
+		}
+	}
 	pod := &core.Pod{}
-	if len(list.Items) == 0 {
+	if len(list.Items) <= 1 {
```

Comment:
if we get here, it necessarily means that there's no conversion pod, no?
the logic in this block of changes seems correct but it's not really readable..

Reply:
yeah, removed the `if` and changed to it be more readable (i hope)